app/src/main/java/com/greenfox/fuchsit/zerdareader/server/MockCall.java

```diff
@@ -0,0 +1,53 @@
+package com.greenfox.fuchsit.zerdareader.server;
+
+import com.greenfox.fuchsit.zerdareader.model.UserResponse;
+
+import java.io.IOException;
+
+import okhttp3.Request;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+import static android.icu.lang.UCharacter.GraphemeClusterBreak.T;
+
+/**
+ * Created by Zsuzska on 2017. 01. 23..
+ */
+
+public class MockCall implements Call<UserResponse> {
```

Comment:
@Anna812 @vikukzs change this to be a general MockCall, so it can have different Response types in it, and also make it abstract so you can remove the enqueue so that will be where this is used:
```java
public abstrac class MockCall<T> implements Call<T> {
...
```